homeworks/vladislav.buzunar_buzyaka/4-objects/objects.js

```diff
@@ -0,0 +1,85 @@
+const tamagochi = {
+    name: 'тамагочi',
+    age: 23,
+    healthLevels: 5,
+    happyLevels: 3,
+    foodLevels: 4,
+    alive: true,
+    get tamagochiStates() {
+        return `Здоров'я  в ${this.name} - ${this.healthLevels} Щастя - ${this.happyLevels} Ситість - ${this.foodLevels}`;
+    },
+    get doGameOver() {
+        return 'Game over!';
```

Comment:
You should not return a string here. Please, console log that game is over and return something else. You can you `this` as a returned value, for example